apps/web/lib/services/blockchain/__tests__/performance.test.ts

```
@@ -0,0 +1,342 @@
+/**
+ * Performance and stress tests for Mock Blockchain Services
+ * Tests service behavior under load and with large datasets
+ */
+
+import { describe, test, expect, beforeEach } from 'bun:test';
+import { MockBlockchainServiceFactory } from '../mock-service-factory';
+import { BadgeType } from '~/lib/types/blockchain/contract-interfaces.types';
```

Biome (2.1.2)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)